bitnami/kafka/templates/statefulset.yaml

```diff
@@ -96,18 +96,17 @@ spec:
           args:
             - -ec
             - |
-              mkdir -p /bitnami/kafka
-              chown -R "{{ .Values.podSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "/bitnami/kafka"
-              mkdir -p /opt/bitnami/kafka/logs
-              chown -R "{{ .Values.podSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "/opt/bitnami/kafka/logs"
-          securityContext:
-            runAsUser: 0
+              chown -R "{{ .Values.podSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "{{ .Values.persistence.mountPath }}"
+              chown -R "{{ .Values.podSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "{{ .Values.logPersistence.mountPath }}"
```

Comment:
I wouldn't remove the `mkdir -p` actions. They're harmless and I've found issues in the past with similar init containers despite volumes being mounted on those specific paths.

Reply:
Actually, if the path does not exist, then the volume isn't mounted (correctly), and the `mkdir` would be hiding the fact that the `chown` doesn't do anything. So adding those back in would mean that it potentially hides a problem with the chart